arch/arm/soc/nxp_lpc/lpc54xxx/Kconfig.soc

```diff
@@ -35,4 +35,23 @@ config SOC_PART_NUMBER_LPC54XXX
 	  option that you should not set directly. The part number selection
 	  choice defines the default value for this string.
 
+config SLAVE_CORE_MCUX
+	bool "Enable LPC54114 Cortex-M0 slave core"
+	default n
+	depends on HAS_MCUX
+	help
+	  Driver for slave core startup
+
+config SLAVE_IMAGE
+	depends on SLAVE_CORE_MCUX
+	string "Binary image of slave core's code"
+
+config SLAVE_BOOT_ADDRESS
```

Comment:
Should SLAVE_BOOT_ADDRESS depend on SLAVE_CORE_MCUX? Or is the intention to be able to use this on slave core too? (currently not used by slave core)

Reply:
It should depend on SLAVE_CORE_MCUX